controllers/issue.js

```diff
@@ -260,8 +260,8 @@ exports.view = function (aReq, aRes, aNext) {
       }
 
       // Discussion
-      discussion = options.discussion = modelParser.parseDiscussion(aDiscussion);
-      modelParser.parseIssue(discussion);
+      options.discussion = discussion = modelParser.parseDiscussion(aDiscussion);
+      modelParser.parseIssue(discussion); // WARNING: Return value never used
```

Comment:
parseDiscussion/parseIssue/parseBlah... will only return a new object if it's still a mongoose Document (with `discussion.toObject()`). Which means the subsequent call to parseIssue won't do that since it's a normal Object. So all the assingments are applied onto the object supplied in the parameter. So no existing bug, just bad practice.

Looks like [I goofed up](https://github.com/Martii/OpenUserJS.org/commit/4ad7a1164566b49106c888b43ccda0fcca20de07#diff-2d78be05ad5a9727f53f0f0d968212bfR168) when I wrote it.

```js
var discussion = modelParser.parseDiscussion(discussionData);
discussion = modelParser.parseIssue(discussion);
options.discussion = discussion;
```